### PR TITLE
feat(FX-4076): improvements for live auction modal

### PR DESCRIPTION
**Pull request by dimatretyak**
This PR resolves [FX-4076] 

### Changes
* Added the ability to close the modal during loading state
* Fixed the problem when title gets truncated

### Demo (title)
| Before | After |
| ------------- | ------------- |
| before | after | 

#### Before
https://user-images.githubusercontent.com/3513494/207865217-b789dde0-da8e-4138-ae97-30d92e628a9f.mp4

#### After
https://user-images.githubusercontent.com/3513494/207865240-0fd634a0-13ad-47c9-90fa-40928a50a9a8.mp4

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

- Improvements for live auction modal - dimatretyak

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[FX-4076]: https://artsyproduct.atlassian.net/browse/FX-4076?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: ios/Artsy/View_Controllers/ARSerifNavigationViewController.m [hunk 1]
```diff
@@ -179,9 +179,10 @@ - (void)navigationController:(UINavigationController *)navigationController will
 
             CGRect labelFrame = label.bounds;
             CGFloat idealWidth = CGRectGetWidth(labelFrame) + xOffset;
+            CGFloat labelHeight = CGRectGetHeight(labelFrame);
             CGFloat max = CGRectGetWidth(navigationController.view.bounds) - (rightButtonsCount * 48) - ((rightButtonsCount - 1) * 10) - 20;
 
-            label.frame = CGRectMake(xOffset, 0, MIN(idealWidth, max), 20);
+            label.frame = CGRectMake(xOffset, 0, MIN(idealWidth, max), MAX(labelHeight, 20));
```
Comment (pvinis) [inline ios/Artsy/View_Controllers/ARSerifNavigationViewController.m:185]: nice

**Review by MounirDhahri — APPROVED** [review]
Nice. LGTM 🙌

**Review by gkartalis — APPROVED** [review]

File: ios/Artsy/View_Controllers/Live_Auctions/LiveAuctionViewController.swift [hunk 2]
```diff
@@ -189,6 +189,9 @@ class LiveAuctionViewController: UIViewController {
     @objc func dismissLiveAuctionsModal() {
         overlaySubscription?.unsubscribe()
         saleOnHoldSubscription?.unsubscribe()
+
+        AREmission.sharedInstance().notificationsManagerModule.requestModalDismiss()
```
[the remark this answers is not in the document]
Comment (brainbicycle) [reply ios/Artsy/View_Controllers/Live_Auctions/LiveAuctionViewController.swift:193]: I think it is probably line 195 we don't need that I meant with the comment? Maybe I removed the wrong line 😬  If it is working we can leave as is but in theory we should only need one of those

**Review by brainbicycle — APPROVED** [review]
🙏 thanks for fixing this!

**Pull request merged**